data/sql/updates/pending_db_auth/rev_1736966800561154000.sql

```diff
@@ -0,0 +1,3 @@
+--
+-- Changes the Length of `username` from 32 to 17 characters as it the client limit.
+ALTER TABLE `account` CHANGE COLUMN `username` `username` VARCHAR(17) NOT NULL DEFAULT '' COLLATE 'utf8mb4_unicode_ci' AFTER `id`;
```

Comment:
Leaving it varchar(32) could have some benefits and save us from headaches.

1. Changing it would error for everyone who for some reason has a username > 17
2. It doesn't really matter as
    - The client would crash
    - Creating a new account/changing username through the console it would send AOR_NAME_TOO_LONG anyways

Reply:
As you and sud suggested, i've removed changes to the column